namespace/builder.go

```diff
@@ -0,0 +1,126 @@
+package namespace
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+// OpenFunc opens files
+type OpenFunc func(path string) (io.Reader, error)
+
+// Builder helps building plan9 namespaces
+type Builder struct {
+	dir  string
+	file File
+
+	open OpenFunc
+}
+
+func open1(path string) (io.Reader, error) { return os.Open(path) }
+
+// NewBuilder returns the default runner
+func NewBuilder() (*Builder, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	return &Builder{
+		dir:  wd,
+		open: open1,
+	}, nil
+}
+
+func newrunner(wd string, b OpenFunc) (*Builder, error) {
```

Comment:
I guess we should go with the go naming conventions for consistency in the code - like newRunner or buildNs